ssm/agents.py

```diff
@@ -53,15 +53,14 @@
 REFRESH_DNS = 600
 
 
-def logging_helper(cp, log_manual_path=''):
+def logging_helper(cp):
     """Take config parser object and set up root logger."""
     try:
-        if os.path.exists(log_manual_path):
```

Comment:
Re-inlcude this line, but add `logging.warning` message, but also note that it needs to be after line 63 where the logging is set up.